pymatgen/entries/compatibility.py

```diff
@@ -477,10 +478,6 @@ def get_correction(self, entry) -> ufloat:
         most_electroneg = elements[-1].symbol
         correction = ufloat(0.0, 0.0)
 
-        # only correct GGA or GGA+U entries
-        if entry.parameters.get("run_type") not in ("GGA", "GGA+U"):
-            return ufloat(0.0, 0.0)
-
```

Comment:
This block should then stay in place, if the above block is deleted

Reply:
Thanks for thorough review @rkingsbury! 👍 

Tip: add `py` to your code fences for syntax highlighting.

````diff
- ```
+ ```py
````